imports/plugins/core/ui-navbar/client/containers/navbar.js

```diff
@@ -19,7 +19,7 @@ function composer(props, onData) {
   }
 
   if (shop && Array.isArray(shop.brandAssets)) {
-    const brandAsset = _.find(this.getShop().brandAssets, (asset) => asset.type === "navbarBrandImage");
```

Comment:
Is there a reason this change was added to this PR? If this is fixing an another issue it should be explicitly called out in the comments. Generally we should keep bugfix PR's pretty focused on fixing just the one issue.

Reply:
I got that change in `brandAssets` when I pulled recent changes from marketplace branch.

Reply:
how would that be possible? If you are up to date with marketplace then why would you have that change? Did you merge a conflict?

Reply:
I made a typographical error in my commit message so I reset my HEAD and unknowingly unstaged the change that was made to the `brandAssets`.